DrawAction/DrawLine.swift

```diff
@@ -63,21 +63,17 @@ final public class DrawLine : DrawAction {
         self.init(startPoint: startPoint, endPoint: endPoint, color: color, lineWidth: 1)
     }
 
-    override func performActionInContext(context: DrawContext) {
+    override func performActionInContext(_ context: DrawContext) {
         context.performGraphicsActions { gContext in
-            CGContextSetLineCap(gContext, .Square)
-            CGContextSetStrokeColorWithColor(gContext, color.CGColor)
-            CGContextSetLineWidth(gContext, lineWidth)
-
-            let rectPoints = pointArrayForUnitPoints(points, inRect: context.rect)
-            rectPoints.withUnsafeBufferPointer { pBuffer in
-                CGContextStrokeLineSegments(gContext, pBuffer.baseAddress, pBuffer.count)
-            }
+            gContext.setLineCap(.square)
+            gContext.setStrokeColor(color.cgColor)
+            gContext.setLineWidth(lineWidth)
+            gContext.strokeLineSegments(between: points)
```

Comment:
you got rid of the call to 'pointArrayForUnitPoints' which translates the normalized points to the current rect. You still need to call that